VideoWeb/VideoWeb/ClientApp/src/app/shared/header/header.component.html

```diff
@@ -4,7 +4,11 @@
 
     <div class="moj-header__logo">
 
-    <app-header-logo-svg></app-header-logo-svg>
+    <img role="presentation" id="header-logo-scot" class="moj-header__logotype-crest" *ngIf="hearingVenueIsScottish$ | async;else ukCrestSvg" src="/assets/images/scot-crest-white.png" height="40" [attr.alt]="'participants-panel.scottish-crest' | translate"/>
+
+    <ng-template #ukCrestSvg>
```

Comment:
```suggestion
    <ng-template #ukCrest>
```
We don't want to give the impression that this has to be an svg